### PR TITLE
cpu_topology_test:Add aarch64 PPTT table check

### PR DESCRIPTION
**Pull request by zhenyzha**
Add aarch64 PPTT table check

ID:2041697
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
git am 3178.patch 
Applying: cpu_topology_test:Add aarch64 PPTT table check

` (1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.cpu_topology_test.arm64-pci: PASS (65.74 s)`

@PaulYuuu Could you pls help review when you are free. thanks.

File: qemu/tests/cfg/cpu_topology_test.cfg [hunk 1]
```diff
@@ -7,3 +7,8 @@
     Linux:
         x86_64:
             check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
+        aarch64:
+            no RHEL.6 RHEL.7 RHEL.8
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            required_qemu = [6.2.0,)
+            check_PPTT_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
```
Comment (PaulYuuu) [inline qemu/tests/cfg/cpu_topology_test.cfg:14]: So, PPTT is not a property, for this checkpoint seems to check the core ID mapping.

Comment (zhenyzha) [reply qemu/tests/cfg/cpu_topology_test.cfg]: Thanks for the reminder, corrected.

File: qemu/tests/cpu_topology_test.py [hunk 2]
```diff
@@ -72,5 +74,8 @@ def check(p_name, exp, check_cmd):
         if not check_if_vm_vcpu_topology_match(session, os_type, vm.cpuinfo):
             test.fail('CPU topology of guest is incorrect.')
         if params.get('check_siblings_cmd'):
-            check('sibling', vcpu_threads * vcpu_cores, 'check_siblings_cmd')
+            check('sibling', vcpu_threads * vcpu_cores, params['check_siblings_cmd'])
+        if params.get('check_core_id_cmd'):
+            for cpu_id in list(range(params['smp'])):
+                check('core_id', cpu_id, params['check_PPTT_cmd'] % cpu_id)
```
Comment (PaulYuuu) [inline qemu/tests/cpu_topology_test.py:80]: Also need to be updated.

Comment (zhenyzha) [reply qemu/tests/cpu_topology_test.py]: Oh sorry for the mistake.
Corrected

git am 3178.patch 
Applying: cpu_topology_test:Add aarch64 PPTT table check

` (1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.cpu_topology_test.arm64-pci: PASS (64.26 s)`

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**